Makefile

```diff
@@ -25,14 +25,27 @@ GIT_COMMIT := $(shell git rev-parse HEAD 2> /dev/null || true)
 
 all: binary
 
-binary:
+binary: ${DEST}skopeo
+
+# Build a docker image (skopeobuild) that has everything we need to build.
+# Then do the build and the output (skopeo) shold appear in ${DEST} dir (or .).
+${DEST}skopeo: cmd/skopeo
+	docker build ${DOCKER_BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
```

Comment:
Am I correct in thinking that the `skopeobuildimage` container does not change on every build? If so, it would be nice to be able to reuse it across builds.